app/assets/javascripts/posts/editor.js

```diff
@@ -381,7 +382,8 @@ function setAliases(aliases, name) {
 
 function setAliasFromID(selectedAliasID) {
   var correctName = $("#character_alias option[value=\""+selectedAliasID+"\"]").text();
-  $("#post-editor .post-character #name").html(correctName);
+  var correctID = $("input#reply_character_id")[0].value
```

Comment:
We can probably pass this as an argument to the function instead; it's available everywhere we call this method from.